src/WebJobs.Script.WebHost/Metrics/LinuxContainerMetricsPublisher.cs

```diff
@@ -27,6 +27,7 @@ public class LinuxContainerMetricsPublisher : IMetricsPublisher
         public const string ContainerNameHeader = "x-ms-functions-container-name";
         public const string HostNameHeader = "x-ms-functions-host-name";
         public const string StampNameHeader = "x-ms-site-home-stamp";
+        public const string ResourceIdHeader = "x-ms-site-arm-id";
```

Comment:
Is the name of this header being dictated by the downstream service, or are we establishing a new header name ourselves here?

Reply:
We are the ones introducing this change in the host as well as the downstream, so this name is up to us.